File: programme/rupee-to-paise/rs-to-p.cs
// Write a program to rupee to paise
using System;

public class RupeeToPaise
{
    public static void Main(string[] args)
    {
        Console.Write("Rupee : ");
        double rupee;
        if (double.TryParse(Console.ReadLine(), out rupee))
        {
            Console.WriteLine("Paise : {0}", rupee * 100);
        } 
        else
        {
            Console.WriteLine("Input Err");
        }
    }
}

File: programme/rupee-to-paise/rs-to-p.go
// Write a program to rupee to paise

package main

import (
	"fmt"
	"strconv"
)

func main() {
	var str string

	fmt.Print("Rupee : ")
	fmt.Scan(&str)

	num, err := strconv.ParseFloat(str, 64)
	if err != nil {
		fmt.Printf("Input error: %s", str)
		return
	}

	fmt.Printf("Paise : %s\n", strconv.FormatFloat(num*100, 'f', -1, 64))
}
